https://bugs.gentoo.org/953122
https://lore.kernel.org/git/20251202-toon-cross-compile-v1-0-cabc8bce529f@iotcl.com/

From 6437cb608c9c9894852499da36e22095adb57a8e Mon Sep 17 00:00:00 2001
Message-ID: <6437cb608c9c9894852499da36e22095adb57a8e.1764817376.git.sam@gentoo.org>
From: Toon Claes <toon@iotcl.com>
Date: Tue, 2 Dec 2025 11:48:08 +0100
Subject: [PATCH 1/3] meson: ignore subprojects/.wraplock

When asking Meson to wrap subprojects, it generates a .wraplock file in
the subprojects/ directory. Ignore this file.

See also https://github.com/mesonbuild/meson/issues/14948.

Signed-off-by: Toon Claes <toon@iotcl.com>
---
 subprojects/.gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/subprojects/.gitignore b/subprojects/.gitignore
index 63ea916ef5..2bb68c8794 100644
--- a/subprojects/.gitignore
+++ b/subprojects/.gitignore
@@ -1 +1,2 @@
 /*/
+.wraplock

base-commit: f0ef5b6d9bcc258e4cbef93839d1b7465d5212b9
prerequisite-patch-id: b59f73a32b9910a2e72eb13797a0f9bd76ce4812
prerequisite-patch-id: 9f1fdad38ca8082e13069df63de6af32599fe654
-- 
2.52.0

